stl/inc/vector

```diff
@@ -1626,6 +1864,7 @@ private:
         _My_data._Orphan_all();
 
         if (_Myfirst) { // destroy and deallocate old array
+            _ASAN_VECTOR_CLEAR;
             _Destroy_range(_Myfirst, _Mylast, _Al);
             _Al.deallocate(_Myfirst, static_cast<size_type>(_Myend - _Myfirst));

```

Comment:
This order is inconsistent with previous occurrences of `_ASAN_VECTOR_CLEAR`, which said: destroy, clear, deallocate.